### DISCUSSION TITLE
About unifying return columns For v4.0

### DISCUSSION THREAD

**Discussion by cvvergara** [pgRouting/pgrouting#2224] [Ideas]
```
For example on dijkstra
pgr_dijkstra(Edges SQL, start_vid,  end_vid  [, directed])
pgr_dijkstra(Edges SQL, start_vid,  end_vids [, directed])
pgr_dijkstra(Edges SQL, start_vids, end_vid  [, directed])
pgr_dijkstra(Edges SQL, start_vids, end_vids [, directed])
pgr_dijkstra(Edges SQL, Combinations SQL [, directed])
RETURNS SET OF (seq, path_seq [, start_vid] [, end_vid], node, edge, cost, agg_cost)
```
Depending on the call, are the columns returned

Change to
```
RETURNS SET OF (seq, path_seq , start_vid, end_vid, node, edge, cost, agg_cost)
```

That way regardless of the call the same columns are returned
That unification of many other official, proposed and experimental functions like pgr_astar, pgr_bdAstar etc.
Having for the same (family) calls the same returned columns, makes it easier to remember, aka less complicated API

**Comment by dkastl** [comment]
And it makes eventually the documentation also easier to understand and maintain.

Right now I feel the documentation grew to a really big extent, and while it's probably not an issue for the users, who lookup a function here and there, unification may help us with maintaining it.
